### ISSUE TITLE
Named parameter supports for PEP 692 

### ISSUE THREAD

**Issue by heejaechang** [microsoft/pylance-release#3868]
```python
class Movie(TypedDict):
    key: str
    key2: int

def foo(**kwargs: Unpack[Movie]):
    pass
    
foo( <= here
```

currently, when user tries to call `foo`, we show only `kwargs` as known parameters for the `foo`, but for this [PEP 692](https://peps.python.org/pep-0692/) case, we not only know the type of the kwargs, but we also know keys (parameter names).

we should improve completion providers to handle this when we show named parameters for the call.

we might try to improve signature help support as well.

**Comment by rchiodo** [comment]
Looks like this was already upvoted in discussions:
https://github.com/microsoft/pylance-release/discussions/3046

**Comment by Salakar** [comment]
Is there any updates for this issue or the discussion? It seems tooling like pylint & mypy now support Unpack for kwargs but completion is one of the things that's still missing 😿

**Comment by rchiodo** [comment]
Funny that you asked, I just started looking at how to implement this like 10 minutes ago. Not sure how long it will take, but probably before next week's prerelease.

**Comment by Salakar** [comment]
> Funny that you asked, I just started looking at how to implement this like 10 minutes ago. Not sure how long it will take, but probably before next week's prerelease.


Thank you so much for looking into it 🎉

**Comment by bschnurr** [comment]
This issue has been fixed in prerelease version 2023.2.31, which we've just released. You can find the changelog here: [CHANGELOG.md](https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#2023231-15-february-2023-prerelease)
